pkg/importer/utils.go

```diff
@@ -83,6 +83,8 @@ func getSyslSafeEndpoint(endpoint string) string {
 	}
 	endpoint = url.PathEscape(endpoint)
 	endpoint = strings.ReplaceAll(endpoint, ".", `%2E`)
+	endpoint = strings.ReplaceAll(endpoint, ":", `%3A`)
+	endpoint = strings.ReplaceAll(endpoint, "+", `%2B`)
```

Comment:
should collect the characters in a map, like `charsToEscape`

Reply:
nice, fixed up!